routers/repo/branch.go

```diff
@@ -80,6 +82,7 @@ func DeleteBranchPost(ctx *context.Context) {
 	defer redirect(ctx)
 	branchName := ctx.Query("name")
 	if branchName == ctx.Repo.Repository.DefaultBranch {
+		log.Warn("DeleteBranch: Can't delete default branch '%s'", branchName)
```

Comment:
I think these should be at Debug - Warn is too high a level for these.

```suggestion
		log.Debug("DeleteBranch: Can't delete default branch '%s'", branchName)
```